### ISSUE TITLE
The bot does not accept the invitation to the quick game.

### ISSUE THREAD

**Issue by franchesko1437** [Levak/warfacebot#322]
hi, Levak.
The bot does not accept the invitation to the quick game.
How to be? he rejects

**Comment by Levak** [comment]
Did you allow game room invitations in your config?
```
wb_postpone_room_invitations 0
wb_accept_room_invitations 1
```

**Comment by franchesko1437** [comment]
Yes. he still rejects

**Comment by seanwlk** [comment]
Are you sure the bot account is allowed to join that room? Rank restriction, mission not unlocked etc...

**Comment by franchesko1437** [comment]
Yes, I am sure. The bot is fine. Once worked, but a very long time in 2018. From now on, all new versions of bots posted on github have a similar problem.

**Comment by franchesko1437** [comment]
https://pp.userapi.com/c849332/v849332793/1a181b/VepHTn1czGA.jpg

**Comment by franchesko1437** [comment]
![image](https://user-images.githubusercontent.com/29465298/58753256-07745900-84c5-11e9-8e60-9f3f23aba5e1.png)

**Comment by Levak** [comment]
Maybe linked to #149  #169

**Comment by franchesko1437** [comment]
These problems have nothing to do with mine. Totally different

**Comment by Levak** [comment]
Should be fixed in d2e8acbeee2d6d021268e51b5d6e5c1063f993e3
New release: https://github.com/Levak/warfacebot/releases/tag/1.0.16
